gui/src/components/maintenance/cortx-about.vue

```diff
@@ -58,6 +58,14 @@
             <label>{{ versionDetails.RELEASE }}</label>
           </td>
         </tr>
+        <tr>
+          <td>
+            <label class="cortx-text-bold">{{$t("aboutUs.SerialNumber")}}</label>
+          </td>
+          <td style="padding-top: 2px;">
```

Comment:
Please if we any class to add padding instead of using inline style, else can you please create a class for this (style="padding-top: 2px;) it used at 3-4 places in this file

Reply:
done